test/image-prepare

```diff
@@ -152,13 +153,23 @@ def build_and_install(build_image, build_results, skips, args):
                                  memory_mb=2048, cpus=4, maintain=True, networking=networking)
     completed = False
 
-    # While the machine is booting, make a tarball
+    # While the machine is booting, make a dist tarball
+
+    # we need to support completely clean git trees, unpacked release tarballs, and already configured trees
+    if not os.path.exists("Makefile"):
+        if os.path.exists('.tarball'):
```

Comment:
why not `./configure` if it exists, otherwise `./autogen.sh`?

Reply:
I just copied that from bots/make-source, but indeed, why not.

Reply:
Done.